Gems/Atom/RHI/DX12/Code/Source/RHI/Device.cpp

```diff
@@ -174,6 +174,8 @@ namespace AZ
             m_limits.m_minConstantBufferViewOffset = D3D12_CONSTANT_BUFFER_DATA_PLACEMENT_ALIGNMENT;
             m_limits.m_maxIndirectDrawCount = static_cast<uint32_t>(-1);
             m_limits.m_maxIndirectDispatchCount = static_cast<uint32_t>(-1);
+            m_limits.m_maxConstantBufferSize = D3D12_REQ_CONSTANT_BUFFER_ELEMENT_COUNT * 4u * 4u; // 4096 vectors * 4 values per vector * 4 bytes per value
+            m_limits.m_maxBufferSize = D3D12_REQ_RESOURCE_SIZE_IN_MEGABYTES_EXPRESSION_C_TERM * (1024u * 1024u); // 2048 MB
```

Comment:
We probably want to make sure that these enums exist in the dx12 header of our min windows version which is (https://en.wikipedia.org/wiki/Windows_10_version_1803). It is from 2018 however there are discussion of increasing it to the one from 2020.

Reply:
you will know if it doesnt as AR will break.

Reply:
Good follow up. AR compiles, so no problems there. Plus these macros already existed in D3D11 and they carried on in D3D12, so they are not something new that could have been added between D3D12 versions.